teaspoons-cli/teaspoons/auth_helper.py

```diff
@@ -42,20 +52,25 @@ def _validate_token(token: str) -> bool:
         # This is just to ensure the token is not expired
         jwt.decode(token, options={"verify_signature": False, "verify_exp": True})
         return True
+    except jwt.ExpiredSignatureError:
+        LOGGER.info("Token expired")
```

Comment:
is this something we want to log to the user as opposed to just taking them to the browser and that whole thing so its just assumed their token is expired

Reply:
fair, we can make this a debug log